### PR TITLE
Fixes and Courses addition

### PR DESCRIPTION
**Pull request by Arouz**

### CODE REVIEW

**Review by maxencelav — DISMISSED** [review]

File: sql-files/steam-lms-generate-db.sql [hunk 1]
```diff
@@ -271,7 +291,7 @@ CREATE TABLE `interfaces_projects` (
   `link` varchar(255) COLLATE utf8mb3_unicode_ci NOT NULL,
   `mode` varchar(20) COLLATE utf8mb3_unicode_ci DEFAULT NULL,
   `is_deleted` tinyint(1) NOT NULL DEFAULT 0,
```
Comment (ultimecreation) [inline sql-files/steam-lms-generate-db.sql:293]: a colon is missaing at the end of line 294 as it is not the last line?
`is_activity_solve` tinyint(1) NOT NULL DEFAULT 0

**Review by ultimecreation — CHANGES_REQUESTED** [review]

File: classroom/assets/js/scripts/displayPanel.js [hunk 2]
```diff
@@ -269,8 +272,11 @@ DisplayPanel.prototype.classroom_table_panel_teacher = function (link) {
                 }
             }
             // Load the classroom with the current cache data
-            let students = getClassroomInListByLink(link)[0].students
-            displayStudentsInClassroom(students, link)
+            // seems to be duplicate call for displayStudentsInClassroom with the code below -> 309 - 310 updated by @Rémi C.
```
[the remark this answers is not in the document]
Comment (Arouz) [reply classroom/assets/js/scripts/displayPanel.js:275]: True i think we should do that in the futur, i will add the date of this comment!

File: classroom/assets/js/scripts/displayPanel.js [hunk 3]
```diff
@@ -344,41 +350,43 @@ DisplayPanel.prototype.classroom_dashboard_new_activity_panel3 = function (ref)
...
-
             } else {
                 ClassroomSettings.activity = id = Number(id.slice(2))
                 Main.getClassroomManager().getOneUserLinkActivity(id).then(function (result) {
                     Activity = result;
                     loadActivityForTeacher();
-                })
+                });
             }
         } else {
```
[the remark this answers is not in the document]
Comment (Arouz) [reply classroom/assets/js/scripts/displayPanel.js:368]: No, i delete this one right now, thank you

File: sql-files/steam-lms-generate-db.sql (hunk 1 above)
Comment (Arouz) [reply sql-files/steam-lms-generate-db.sql:293]: Yes you're right, i did not saw this one ! thank you

**Review by ultimecreation — APPROVED** [review]

**Pull request merged**